config.js

```diff
@@ -0,0 +1,1590 @@
+const x = {
+  mode: 'development',
+  context: '/home/evan/Development/DTI/DTI-NOVA/dti-nova',
+  node: {
+    setImmediate: false,
+    process: 'mock',
+    dgram: 'empty',
+    fs: 'empty',
+    net: 'empty',
+    tls: 'empty',
+    child_process: 'empty'
+  },
+  output: {
+    path: '/home/evan/Development/DTI/DTI-NOVA/dti-nova/dist',
+    filename: 'js/[name].js',
+    publicPath: '/',
+    chunkFilename: 'js/[name].js'
+  },
```

Comment:
@rockon999 🤔

Reply:
`vue-cli-service` uses an internal webpack config now. I dumped it to figure out some plugin issues, and forgot I dumped it.